From fd7c284945df3402380b21f2bfacff6ee3354528 Mon Sep 17 00:00:00 2001
From: Vince Perri <viperri@microsoft.com>
Date: Sat, 5 Mar 2022 00:09:48 +0000
Subject: [PATCH 13/15] net/mlx5: E-Switch, Increase supported number of

>From d40bfeddacd6b4c05a28e3e8f7cb18989a4e134f Mon Sep 17 00:00:00 2001
>From: Maor Dickman <maord@nvidia.com>
>Date: Sun, 5 Sep 2021 14:22:11 +0300
>Subject: [PATCH 13/13] net/mlx5: E-Switch, Increase supported number of
> forward destinations to 32
>
>Increase supported number of forward destinations in the same rule, local
>and remote, from 2 to 32.
>
>Signed-off-by: Maor Dickman <maord@nvidia.com>
>Reviewed-by: Roi Dayan <roid@nvidia.com>
>Signed-off-by: Saeed Mahameed <saeedm@nvidia.com>
---
 drivers/net/ethernet/mellanox/mlx5/core/eswitch.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/mellanox/mlx5/core/eswitch.h b/drivers/net/ethernet/mellanox/mlx5/core/eswitch.h
index 2c7444101..afa4d11ea 100644
--- a/drivers/net/ethernet/mellanox/mlx5/core/eswitch.h
+++ b/drivers/net/ethernet/mellanox/mlx5/core/eswitch.h
@@ -433,7 +433,7 @@ enum mlx5_flow_match_level {
 };
 
 /* current maximum for flow based vport multicasting */
-#define MLX5_MAX_FLOW_FWD_VPORTS 2
+#define MLX5_MAX_FLOW_FWD_VPORTS 32
 
 enum {
 	MLX5_ESW_DEST_ENCAP         = BIT(0),
-- 
2.17.1

